pdfly/cat.py

```diff
@@ -87,8 +91,15 @@ def main(
                     f"WARNING: Page range {page_range} is out of bounds",
                     file=sys.stderr,
                 )
-            for page_num in range(*page_range.indices(len(reader.pages))):
-                writer.add_page(reader.pages[page_num])
+            if not use_complements:
+                for page_num in range(*page_range.indices(len(reader.pages))):
+                    writer.add_page(reader.pages[page_num])
+            else:
```

Comment:
You could invert the `if` / `else` to avoid the negation with `not`, that could make the code slightly easier to read 🙂